### ISSUE TITLE
Bug: iLVL display wrong number on toast

### ISSUE THREAD

**Issue by Ghaan** [ls-/ls_Toasts#67]
![bild](https://github.com/user-attachments/assets/657de988-4b16-4e74-9737-935fc70c6ed4)

As you can see here it is displaying the wrong ilvl on the toast itself vs what ilvl the item actually has.

**Comment by Taizunz** [comment]
Just here to confirm this.
![image](https://github.com/user-attachments/assets/4fc34c43-3944-438f-b6ef-30b0c2176919)

**Comment by ls-** [comment]
It's a known bug. Blizz have 3 APIs that can return an item level, I use the second best one, but I can't use the most precise one. Potentially, I could scan tooltips, but it can be rather laggy and that's not something I'm fond of, but I may change it later.

**Comment by ls-** [comment]
I reported this issue to Blizz yet again, but this time around via a trusty comrade who some contacts on the inside, so maybe they'll finally fix this bs 😅 For now we can only hope 🙏🏼

**Comment by Ghaan** [comment]
If that lead doesn't pan out, I know the addon Baganator had a similar issue that ended up being resolved. Can't say in which way, but may be worth looking into if the report to Blizz doesn't work out.

https://github.com/Baganator/Baganator/issues/169

**Comment by ls-** [comment]
Yeah, I'm aware of that API, that's what I meant when I said that I was using the second best option. The best option is what baganator uses, but it's not something that can be used for toasts. 

The only accurate alternative to that is parsing tooltips which I'd like to avoid because it comes with its own host of issue.
